common/views/components/Divider/Divider.js

```diff
@@ -0,0 +1,5 @@
+import {withModifiers} from '../../../utils/classnames';
+
+export default ({modifiers, extraClasses = []}) => (
+  <hr className={`${withModifiers('divider', modifiers)} ${extraClasses.join(' ')}`} />
```

Comment:
Not sure I'm a fan of `withModifiers`for multiple reason - 1 being static code analysis, 2 being they are just classes, why create this abstraction?

Reply:
I agree, this just replicates what we had pretty much, but was chatting to @davidpmccormick last week about how this is an over abstraction. Good opportunity to remove this.

Reply:
If we're happy to be doing a bit of a sweep up while we're at it, it could be good.